client/__tests__/e2e/puppeteer.setup.js

```diff
@@ -17,7 +17,9 @@ setDefaultOptions({ timeout: 20 * 1000 });
 
 jest.retryTimes(ENV_DEFAULT.RETRY_ATTEMPTS);
 
-(async () => {
+beforeEach(async () => {
+  await jestPuppeteer.resetBrowser();
```

Comment:
Adds `resetBrowser()`, so Auth0 session is not persisted from one test to another, which breaks Jest auto-retries and other tests that rely on logging in before assertion